Workload-Security/Support/ReadMe.md

```diff
@@ -0,0 +1,49 @@
+# Trend Cloud One Workload Security - Agent Diagnostic Collection Tool
```

Comment:
I would suggest to keep the name of the `readme` like the others projects `README.md`

Reply:
FIxed!